### PR TITLE
Add oc_output files to must gather test lists

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/must_gather/const_must_gather.py [hunk 1]
```diff
@@ -168,13 +168,46 @@
...
+    "get_nodes_-o_wide_--show-labels",
+    "desc_clusterversion",
+    "get_pv",
+    "desc_sc",
+    "desc_infrastructures.config",
+    "csv",
+    "rolebinding",
+    "all",
+    "role",
+    "storagecluster.yaml",
```
Comment (crombus) [inline ocs_ci/ocs/must_gather/const_must_gather.py:210]:
```all                  events               pods                 role                 storagecluster       subscription         
csv                  installplan          pods_-owide          rolebinding          storagecluster.yaml  volumesnapshotclass  ```

storagecluster --> describe and volumesnapshotclass missing.

**Review by crombus — CHANGES_REQUESTED** [review]
looks good. Few files left

**Comment by crombus** [comment]
Do you follow certain format for commits. I mean to ask add `must-gather:` in front of commit msg.

**Review by ebenahar — APPROVED** [review]

**Comment by OdedViner** [comment]
Test pass on my pc:
```
-------------------------------- live log call ---------------------------------
00:21:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get  csv -n openshift-storage -o yaml
00:21:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get Pod  -n openshift-storage -o yaml
PASSED                                                                   [ 33%]
------------------------------ live log teardown -------------------------------
00:21:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
00:21:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
00:21:39 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-6994f49bdd-bz9hs -- ceph health
00:21:41 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
00:21:41 - MainThread - tests.conftest - INFO - Ceph health check passed at teardown

tests/manage/z_cluster/test_must_gather.py::TestMustGather::test_must_gather[JSON] 
-------------------------------- live log setup --------------------------------
00:21:41 - MainThread - tests.conftest - INFO - Checking for Ceph Health OK 
00:21:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
00:21:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
00:21:43 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-6994f49bdd-bz9hs -- ceph health
00:21:45 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
00:21:45 - MainThread - tests.conftest - INFO - Ceph health check passed at setup
-------------------------------- live log call ---------------------------------
00:21:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get  csv -n openshift-storage -o yaml
00:21:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get Pod  -n openshift-storage -o yaml
PASSED                                                                   [ 66%]
------------------------------ live log teardown -------------------------------
00:22:02 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
00:22:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
00:22:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-6994f49bdd-bz9hs -- ceph health
00:22:05 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
00:22:05 - MainThread - tests.conftest - INFO - Ceph health check passed at teardown

tests/manage/z_cluster/test_must_gather.py::TestMustGather::test_must_gather[OTHERS] 
-------------------------------- live log setup --------------------------------
00:22:05 - MainThread - tests.conftest - INFO - Checking for Ceph Health OK 
00:22:05 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
00:22:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
00:22:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-6994f49bdd-bz9hs -- ceph health
00:22:09 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
00:22:09 - MainThread - tests.conftest - INFO - Ceph health check passed at setup
-------------------------------- live log call ---------------------------------
00:22:09 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get  csv -n openshift-storage -o yaml
00:22:19 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get  csv -n openshift-storage -o yaml
00:22:21 - MainThread - ocs_ci.ocs.must_gather.must_gather - INFO - Verify noobaa_diagnostics folder exist
00:22:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get Pod  -n openshift-storage -o yaml
PASSED                                                                   [100%]```

File: ocs_ci/ocs/must_gather/const_must_gather.py (hunk 1 above)
Comment (OdedViner) [reply ocs_ci/ocs/must_gather/const_must_gather.py:210]: I will add volumesnapshotclass, storagecluster

**Review by ebenahar — APPROVED** [review]

**Review by shreekarSS — APPROVED** [review]

**Pull request merged**